src/pyOpenMS/pxds/FIAMSDataProcessor.pxd

```diff
@@ -16,14 +16,52 @@ cdef extern from "<OpenMS/ANALYSIS/ID/FIAMSDataProcessor.h>" namespace "OpenMS":
         #
         # wrap-inherits:
         #  DefaultParamHandler
-        FIAMSDataProcessor() nogil except +
-        FIAMSDataProcessor(FIAMSDataProcessor) nogil except +
+        FIAMSDataProcessor() nogil except + # wrap-doc:Data processing for FIA-MS data
+        FIAMSDataProcessor(FIAMSDataProcessor) nogil except + # wrap-ignore
         bool run(MSExperiment & experiment, float & n_seconds, MzTab & output, bool load_cached_spectrum) nogil except +
+            # wrap-doc:
+                #   Run the full analysis for the experiment for the given time interval
+                #   -----
+                #   The workflow steps are:
+                #   - the time axis of the experiment is cut to the interval from 0 to n_seconds
+                #   - the spectra are summed into one along the time axis with the bin size determined by mz and instrument resolution
+                #   - data is smoothed by applying the Savitzky-Golay filter
+                #   - peaks are picked
+                #   - the accurate mass search for all the picked peaks is performed
+                #
+                #   The intermediate summed spectra and picked peaks can be saved to the filesystem. 
+                #   Also, the results of the accurate mass search and the signal-to-noise information 
+                #   of the resulting spectrum is saved.
+                #   -----
+                #   :param experiment: Input MSExperiment
+                #   :param n_seconds: Input number of seconds
+                #   :param load_cached_spectrum: Load the cached picked spectrum if exists
+                #   :param output: [out] Output of the accurate mass search results
+                #   :return: a boolean indicating if the picked spectrum was loaded from the cached file
```

Comment:
Is there a difference between when to use `:return:` and when `:returns:`. I have seen both in reST format examples?

Reply:
Maybe again start with a capital letter.

Reply:
No difference. I have left 'a' small intentionally.